src/app.ts

```diff
@@ -39,18 +40,25 @@ Downloader.downloadFile(crawlBaseUrl + fileName, dataFolder, (err, filepath) =>
             console.log("unpacking complete!");
         }
 
-        // TODO: Language filter
+        // Extract english pages
+        LanguageExtractor.extractWETPages("./data/CC-MAIN-20170116095119-00016-ip-10-171-10-70.ec2.internal.warc.wet", 'eng', (err,filepath) => {
```

Comment:
Remove the file path. `"./data/CC-MAIN-20170116095119-00016-ip-10-171-10-70.ec2.internal.warc.wet"`,
`filepath` already contains the path for the downloaded and unpacked file. Just replace it with `filepath` ;)